google-cloud-pubsub/lib/google/cloud/pubsub/topic/async_publisher.rb

```diff
@@ -111,10 +161,18 @@ def flush
             self
           end
 
+          ##
+          # Checks if the publisher has been started.
+          #
+          # @return [boolean] `true` when started, `false` otherwise.
           def started?
             !stopped?
           end
 
+          ##
+          # Checks if the publisher has been stopped.
```

Comment:
"Checks if" suggests to me that it makes an RPC. How about "Whether the..." instead for these booleans?